galaxy/tools/deps/conda_util.py

```diff
@@ -54,7 +55,8 @@ class CondaContext(installable.InstallableContext):
 
     def __init__(self, conda_prefix=None, conda_exec=None,
                  shell_exec=None, debug=False, ensure_channels='',
-                 condarc_override=None, use_path_exec=USE_PATH_EXEC_DEFAULT, copy_dependencies=False):
+                 condarc_override=None, use_path_exec=USE_PATH_EXEC_DEFAULT,
+                 use_local=USE_LOCAL_DEFAULT, copy_dependencies=False):
```

Comment:
I think I'd add `use_local` as last parameter (at least for consistency with how you call it in `galaxy/tools/deps/resolvers/conda.py`)